libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -789,6 +797,7 @@ - (void)handleUserDidSwitch:(SFUserAccount *)fromUser toUser:(SFUserAccount *)to
 {
     [SFSecurityLockout setupTimer];
     [SFSecurityLockout startActivityMonitoring];
+    [self swizzlePasteboardIfNeeded];
```

Comment:
Same question. Disabling cut&paste is a managed preference for app. So why do we need to check here again?

Reply:
It's configured on the connected app so it could change between users on different orgs, this would check if it needs to be changed to the named pasteboard or to the general one depending on the current state and the setting